CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+cmake_minimum_required(VERSION 3.5)
+project(Maya2glTF)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/modules")
+
+find_package(Maya REQUIRED)
+
+include(ExternalProject)
+
+# COLLADA2GLTF
+ExternalProject_Add(COLLADA2GLTF
+  GIT_REPOSITORY https://github.com/KhronosGroup/COLLADA2GLTF.git
+  GIT_TAG v2.1.3
+  PREFIX COLLADA2GLTF
+  INSTALL_DIR
+  CMAKE_ARGS
+	-DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
```

Comment:
If you change this to:

```cmake
CONFIGURE_COMMAND ${CMAKE_COMMAND} -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR> <SOURCE_DIR>/GLTF
```

It will only build the glTF library so that you don't have to build the rest of COLLADA2GLTF + OpenCOLLADA.

Reply:
Thanks! 

I'm really struggling with CMAKE, I guess I will first need to RTFM 😉 

Right now I'm trying to figure out the correct way to link the Maya2glTF DLL. In your PR it was configured as a static lib, but it needs to be shared DLL (you already mentioned it was broken, I guess that was the reason). This gives tons of linker errors of course. I need to add gltf, draco, and the Microsoft CRT library somehow.